lessons/pydata/homework_revisited/index.ipynb

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
<p>Nerozumím "<em>Řada programátorů toto používá i k označování proměnných v kódu." Jako myslí se, že obecně proměnných, nebo proměnných v našem případě? (asi to druhé)</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>